src/misc/news/NewsListItem.ts

```diff
@@ -11,7 +11,7 @@ export interface NewsListItem {
 	render(newsId: NewsId): Children
 
 	/**
-	 * Return true iff the news should be shown to the logged-in user.
```

Comment:
this was probably not a typo: https://en.wikipedia.org/wiki/If_and_only_if (the other direction is kind of implied though)

with the new doc comment, we'd be allowed to return `true` always no matter if the news item should be shown or not.

Reply:
oh, my bad :facepalm: